WooCommerce/src/main/kotlin/com/woocommerce/android/push/WooNotificationBuilder.kt

```diff
@@ -0,0 +1,242 @@
+package com.woocommerce.android.push
+
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.content.ContentResolver
+import android.content.Context
+import android.content.Intent
+import android.graphics.Bitmap
+import android.media.AudioAttributes
+import android.media.AudioManager
+import android.net.Uri
+import android.os.Build
+import android.os.RemoteException
+import androidx.core.app.NotificationCompat
+import androidx.core.app.NotificationManagerCompat
+import androidx.core.content.ContextCompat
+import com.bumptech.glide.Glide
+import com.woocommerce.android.R
+import com.woocommerce.android.model.Notification
+import com.woocommerce.android.ui.main.MainActivity
+import com.woocommerce.android.util.WooLog
+import org.wordpress.android.util.ImageUtils
+import org.wordpress.android.util.PhotonUtils
+import java.io.UnsupportedEncodingException
+import java.net.URLDecoder
+import java.util.concurrent.ExecutionException
+import javax.inject.Singleton
+
+@Suppress("TooManyFunctions")
+@Singleton
+class WooNotificationBuilder constructor(private val context: Context) {
```

Comment:
🔧 To simplify DI setup, can we remove `NotificationModule` and use constructor injection here?

```suggestion
class WooNotificationBuilder @Inject constructor(private val context: Context) {
```

Reply:
Good point! Thanks for the suggestion! Fixed in 259e257